mobx_codegen/lib/src/template/store.dart

```diff
@@ -8,35 +8,38 @@ import 'package:mobx_codegen/src/template/observable_stream.dart';
 import 'package:mobx_codegen/src/template/params.dart';
 import 'package:mobx_codegen/src/template/rows.dart';
 
-class StoreTemplate {
-  final SurroundedCommaList<TypeParamTemplate> typeParams =
-      SurroundedCommaList('<', '>', []);
-  final SurroundedCommaList<String> typeArgs =
-      SurroundedCommaList('<', '>', []);
-  String mixinName;
-  String parentName;
+class SubclassStoreTemplate extends StoreTemplate {
+  String get typeName => publicTypeName;
 
-  final Rows<ObservableTemplate> observables = Rows();
-  final Rows<ComputedTemplate> computeds = Rows();
-  final Rows<ActionTemplate> actions = Rows();
-  final Rows<AsyncActionTemplate> asyncActions = Rows();
-  final Rows<ObservableFutureTemplate> observableFutures = Rows();
-  final Rows<ObservableStreamTemplate> observableStreams = Rows();
+  @override
+  String toString() => '''
```

Comment:
this comment string should be pulled out as a static constant in `StoreTemplate`

Reply:
Done